dependencies/src/main/java/com/google/cloud/tools/opensource/dependencies/DependencyGraph.java

```diff
@@ -79,6 +81,9 @@
   // map of groupId:artifactId:version to paths
   private SetMultimap<String, DependencyPath> paths = HashMultimap.create();
 
+  private final ListMultimap<DependencyPath, DependencyPath> parentToChildren =
+      LinkedListMultimap.create();
```

Comment:
do we need to preserve insertion order here? If so, the type should be LinkedListMultimap